integration-tests/resteasy-jackson/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 message=Production
+quarkus.smallrye-openapi.store-schema-directory=target/generated
```

Comment:
I don't understand why you need to define that one? Wouldn't it be the default now?

Reply:
Hi @gsmet - default is not storing the document at all, like it is currently.